venter/models.py

```diff
@@ -47,11 +47,11 @@ class Complaints(models.Model):
     tags = models.ManyToManyField(TagUris, related_name='tags', blank=True)
     users_up_voted = models.ManyToManyField('users.UserProfile', related_name='users_up_voted', blank=True)
     authorities = models.ManyToManyField(Authorities, related_name='complaints', blank=True)
-    email_status = models.BooleanField(default=False)
+    email_sent_to = models.CharField(max_length=50, blank=True, null=True)
```

Comment:
50 is too low; complaints might have several emails. Change this to a `TextField` instead.